docs/Changelog.md

```diff
@@ -10549,6 +10549,62 @@ This version of the operator has been available since version 11 of the default
 <dd>tensor of bool, which should be a scalar.</dd>
 </dl>
 
+### <a name="RoiCropAndResize-11"></a>**RoiCropAndResize-11**</a>
+
+  Extracts crops from the input image tensor and resizes them using bilinear sampling or nearest neighbor sampling
+  (possibly with aspect ratio change) to a common output size specified by crop_height and crop_width.
+  Returns a tensor with crops from the input image at positions defined at the bounding box locations in boxes.
+  The cropped boxes are all resized (with bilinear or nearest neighbor interpolation) to
+  a fixed size = [crop_height, crop_width]. The result is a 4-D tensor [num_boxes, crop_height, crop_width, depth].
+  The resizing is corner aligned. Suppose the input tensor X of shape (N, C, H, W), and the input rois tensor R of shape (M, 4),
+  and the batch indices tensor B of shape (M, ). The shape of output tensor Y is (M, C, CH, CW) where CH and CW is the crop_height
```

Comment:
Add equation or pseudocode?

Reply:
I have the python implementation [here](https://github.com/onnx/onnx/pull/2142/files/4f82d4f9f37e8980e29320fc20cc1f2a7026e412#diff-bd6ef8521750dfb968d157511646c345), which is very lengthy. Here I already mentioned `Y[m, c, ch, cw] = mode(X[B[m], c, ch_x, cw_x], extrapolation_value)`, it is clear now.